dojo/tools/wazuh/parser.py

```diff
@@ -0,0 +1,82 @@
+import json
+from dojo.models import Finding
+
+
+class WazuhParser(object):
+    """
+    Use Wazuh Vulnerability API to retrieve the findings
+    The vulnerabilities with condition "Package unfixed" are skipped because there is no fix out yet.
+    https://github.com/wazuh/wazuh/issues/14560
+    """
+
+    def get_scan_types(self):
+        return ["Wazuh"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "Wazuh"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Wazuh"
+
+    def get_findings(self, filename, test):
+        data = json.load(filename)
+        # Detect duplications
+        dupes = dict()
+
+        try:
+            vulnerability = data[next(iter(data.keys()))]["affected_items"]
+        except (KeyError, StopIteration):
+            return list()
+
+        if vulnerability is None:
+            return list()
+
+        for item in vulnerability:
+            if item['condition'] != "Package unfixed" and item['severity'] != "Untriaged":
```

Comment:
this conditional branch puzzle me.
Does "Untriaged" means we don't know the severity?
If yes, we should convert it to `Info`

Reply:
"Untriaged" can be translated with "pending". This is why I had it classified already as an "info" finding in this PR. But I like it this way better because severities of findings are not adjusted afterwards in DefectDojo once they are stored there. The following fix could help to first use "Info" and then later adjust the severity of the finding to something besides "Info" as soon as the severity is not "untriaged" anymore.
https://github.com/DefectDojo/django-DefectDojo/issues/6774

Reply:
Does this answer your question @damiencarol ? 
Let's leave it like this now as then a finding gets reported in the right severity as soon as the finding is not Untriaged anymore.